### DISCUSSION TITLE
Option to hide Open API Client Button

### DISCUSSION THREAD

**Discussion by markwinter** [scalar/scalar#4042] [Ideas]
Can we get an option to hide the Open API Client button please? I don't want users interacting with the API via their browser. It also just doesn't work as our oapi yaml is on a private network

**Comment by antlio** [comment]
hey @markwinter thanks for the request, a pr has been open in order to enable this:
https://github.com/scalar/scalar/pull/4047

**Reply by markwinter** [reply]
Thanks!
